src/modules/IscsiLioData.rb

```diff
@@ -599,7 +599,8 @@ def GetIpAddr
 
       ip_list.reject! do |address|
         address.start_with?("127.") ||  # local IPv4
-          address.start_with?("::1") # local IPv6
+          address.start_with?("::1") || # local IPv6
+          address.start_with?("fe80:")  # scope link IPv6
       end
 
       ip_list = [""] if ip_list.empty?
```

Comment:
It's because directly used in ComboBox (dialogs.rb). I will add documentation.
ComboBox(
                      Id(:ipaddr),
                      Opt(:hstretch),
                      _("Ip address"),
                      IscsiLioData.GetIpAddr